blueocean-core-js/src/js/paths/rest.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This object defines rest paths
+ */
+export default {
+    apiRoot() {
+        return '/blue/rest';
+    },
+
+    organizationPipelines(organizationName) {
+        return `${this.apiRoot()}/search/?q=type:pipeline;organization:${encodeURIComponent(organizationName)};excludedFromFlattening:jenkins.branch.MultiBranchProject,hudson.matrix.MatrixProject&filter=no-folders`;
+    },
+
+    allPipelines() {
+        return `${this.apiRoot()}/search/?q=type:pipeline;excludedFromFlattening:jenkins.branch.MultiBranchProject,hudson.matrix.MatrixProject&filter=no-folders`;
+    },
+
+    activities(organization, pipeline) {
+        return `${this.apiRoot()}/organizations/${organization}/pipelines/${pipeline}/activities/`;
+    },
```

Comment:
This will break for nested jobs if the value of `pipeline` isn't actually the `fullName` so probably wise to rename it so that others don't fall into the same trap. Also the HAL hrefs actually a URL in this format for nested jobs:

`/blue/rest/organizations/jenkins/pipelines/folder1/pipelines/folder2/pipelines/folder3/pipelines/folder4/pipelines/nested-freestyle/activities/`

I *think* you can get away with omitting the `/pipelines` delimiter  but that might be a bug or unintended behavior. I think this file should build HREFs that comply with the metadata and has some test coverage.